modules/EmailMan/EmailMan.php

```diff
@@ -1033,61 +1096,51 @@ public function sendOptInEmail(EmailAddress $emailAddress, $type, $id) {
                 return false;
             }
```

Comment:
Incomplete logic, needs an `else` here to handle when the `$focus` not been returned by any reason above at Bean Factory so we still possible to not having Email address GUID at this point.

Reply:
ok